cheddar/cheddar-integration-aws/build.gradle

```diff
@@ -2,21 +2,22 @@ apply from: '../../test.gradle'
 apply from: '../../logging-api.gradle'
 
 def springVersion = '4.1.1.RELEASE'
+def awsSdkVersion = '1.11.6'
```

Comment:
Note this will cause breaking changes in our applications (as they use `com.amazonaws.util.json.JSONObject` and others) which was removed from the SDK in version 1.11.0. They will need to be updated to use `com.amazonaws.util.json.Jackson` or simply use Jackson directly.